node.gyp

```diff
@@ -970,8 +970,8 @@
             'src/crypto/crypto_tls.h',
             'src/crypto/crypto_clienthello.h',
             'src/crypto/crypto_context.h',
-            'src/crypto/crypto_ecdh.h',
             'src/crypto/crypto_hkdf.h',
+            'src/crypto/crypto_ec.h',
```

Comment:
Nit:
```suggestion
            'src/crypto/crypto_ec.h',
            'src/crypto/crypto_hkdf.h',
```

I don't think the ordering has to change?

Reply:
I really want to `: sort -u` 😢